configs/crashtests-webrtc.ini

```diff
@@ -0,0 +1,28 @@
+[runtests]
+# Settings related to executing runtestsremote.py
+
+# test_name is a descriptor used by runtestsremote.py to
+# determine which of the downloaded unit tests to run.
+#
+# test_manifest is the path to the location of the test_manifest.
+#
+# test_package_names is a list of the keys for test_packages.json
+# which will need to be downloaded in order to run the test.
+
+test_name = crashtest
+test_manifest = reftest/tests/testing/crashtest/autophone-webrtc.list
+test_package_names = reftest
+
+unittest_defaults = configs/unittest-defaults.ini
+
+# How many times to run the tests per phone.
+iterations = 1
+
+# How many chunks for the test
+total_chunks = 1
+
+[treeherder]
+job_name = Autophone Crashtest
+job_symbol = Cw
```

Comment:
Is Cwrtc too long? Maybe Cwrt ? Just want to make sure we don't have a better candidate to use Cw as the symbol.